ext_conf_template.txt

```diff
@@ -0,0 +1,2 @@
+# cat=modules//; type=string; label= Path to (protected) upload folder. : Example '3:/user_upload/userimport/' where the id (3) has to match the uid of your (protected) storage record.
+uploadStorageFolder = 3:/user_upload/userimport/
```

Comment:
@mindpilot Setting a fictitious default (folder `user_upload/userimport` in Storage with uid 3) is not a good practise IMO. It implies it works out of the box, but it won't, because both the storage and the folder need to be there. Of course this is mentioned in the readme, but still.
I suggest: Empty default, but keep the example and throw an exception in the `UploadedFileReferenceConverter` if the value is not set.